app/views/pages/index.html.erb

```diff
@@ -13,63 +13,63 @@
           <table id="datatable">
             <tbody>   
               <tr>
-                  <td style="padding-left: 40px;"><p style="font-size: 18px">Nombre:</p></td>
-                  <td><p style="font-size: 18px"><%= current_user.name %></p></td>
+                <td style="padding-left: 40px;"><p style="font-size: 18px">Nombre:</p></td>
```

Comment:
I wonder if this should be in a different view and render it accordingly since it is duplicated, probably not related to your PR, we can chat offline.

Reply:
Definitely, but there is plenty of repeated code across the codebase. I think that we can create a task just to remove all the duplicated code in the project, before starting to introduce new features.